docker-compose.yml

```diff
@@ -5,11 +5,11 @@ server:
   volumes:
    - .:/src
    - /var/run/docker.sock:/tmp/docker.sock
-   - ./config.yml:/config.yml
+   - ~/volumes/ci/roger:/roger/
```

Comment:
Yes, alternatives remembering the problem of not being able to mount a file just a directory on boot2docker deployments (Mac/Win)
* Create a directory in home folder..
* Provide different docker-compose.yml's one *nix one mac/win
* Users have to change a config file and rebuild the image

More?